cachito/workers/tasks/celery.py

```diff
@@ -22,6 +23,7 @@
 
 
 celeryd_init.connect(validate_celery_config)
+celeryd_init.connect(populate_ssh_known_hosts)
```

Comment:
I'm wondering if this is gonna work when we deploy it to OCP. The `~/.ssh` directory is not writable in the pods.

Reply:
You can set the `core.sshCommand` config to use a ssh config location of your choosing. https://stackoverflow.com/a/41219960